### PR TITLE
TELCODOCS-1564 - RNs - Known Issues - Telco RAN 4.15

### PR DESCRIPTION
**Pull request by slovern**
Version(s): 4.15

Issue: https://issues.redhat.com/browse/TELCODOCS-1564

Link to docs preview: https://71535--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-known-issues

QE review:
- [x] QE has approved this change.

QE approvals are as follows:
- https://issues.redhat.com/browse/OCPBUGS-17422 - [hhassid](https://github.com/hhassid)
- https://issues.redhat.com/browse/OCPBUGS-19838 - [hhassid](https://github.com/hhassid)
- https://issues.redhat.com/browse/RHEL-9279 - [linuxqiao](https://github.com/linuxqiao)
-https://issues.redhat.com/browse/RHEL-9148 - [linuxqiao](https://github.com/linuxqiao)

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -1694,6 +1694,25 @@ In the following tables, features are marked with the following statuses:
 
 * When running a cluster with FIPS enabled, you might receive the following error when running an older version of the OpenShift CLI (`oc`) on a {op-system-base} 9 system: `FIPS mode is enabled, but the required OpenSSL backend is unavailable`. As a workaround, use the `oc` binary provided with the {product-title} cluster. (link:https://issues.redhat.com/browse/OCPBUGS-23386[*OCPBUGS-23386*])
 
+* When you run CNF latency tests on an {product-title} cluster, the `oslat` test can sometimes return results greater than 20 microseconds. This results in an `oslat` test failure.
```
Comment (slovern) [inline release_notes/ocp-4-15-release-notes.adoc:1697]: @linuxqiao Can you provide lgtm on this OCP 4.15 known issue description for https://issues.redhat.com/browse/RHEL-9279 please
This is a direct copy of the 4.14 Knonw Issues note

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -1694,6 +1694,25 @@ In the following tables, features are marked with the following statuses:
...
+(link:https://issues.redhat.com/browse/RHEL-9279[*RHEL-9279*])
+
+* When you use `preempt-rt` patches with the realtime kernel and you update the SMP affinity of a network interrupt, the corresponding IRQ thread does not immediately receive the update.
```
Comment (slovern) [inline release_notes/ocp-4-15-release-notes.adoc:1700]: Also this 4.15 known issue - @linuxqiao 
Again this is a direct copy of the 4.14 known issue note for https://issues.redhat.com/browse/RHELPLAN-138733 please

Comment (slovern) [inline release_notes/ocp-4-15-release-notes.adoc:1700]: Also this one @linuxqiao
Again a direct copy from 4.14 to 4.15 
for https://issues.redhat.com/browse/RHEL-9148

Comment (linuxqiao) [reply release_notes/ocp-4-15-release-notes.adoc:1702]: /lgtm
These issues have not yet been resolved

File: release_notes/ocp-4-15-release-notes.adoc (hunk 1 above)
Comment (linuxqiao) [reply release_notes/ocp-4-15-release-notes.adoc:1699]: /lgtm
These issues have not yet been resolved

**Comment by slovern** [comment]
Sending for peer review - the build errors are caused by the security settings of two bugs. Aiming to get those security settings adjusted, or failing that will have to remove the offending bugs before merging.

File: release_notes/ocp-4-15-release-notes.adoc [hunk 3]
```diff
@@ -1696,6 +1696,25 @@ In the following tables, features are marked with the following statuses:
 
 * In {product-version} with IPv6 networking running on {rh-openstack-first} environments, `IngressController` objects configured with the `endpointPublishingStrategy.type=LoadBalancerService` YAML attribute will not function correctly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2263550[*BZ#2263550*], link:https://bugzilla.redhat.com/show_bug.cgi?id=2263552[*BZ#2263552*])
 
+* When you run CNF latency tests on an {product-title} cluster, the `oslat` test can sometimes return results greater than 20 microseconds. This results in an `oslat` test failure.
```
Comment (xenolinux) [inline release_notes/ocp-4-15-release-notes.adoc:1699]:
```suggestion
* When you run Cloud-native Network Functions (CNF) latency tests on an {product-title} cluster, the `oslat` test can sometimes return results greater than 20 microseconds. This results in an `oslat` test failure.
```

**Review by xenolinux — COMMENTED** [review]
A few suggestions to consider; otherwise looks good

/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

File: release_notes/ocp-4-15-release-notes.adoc [hunk 4]
```diff
@@ -1696,6 +1696,25 @@ In the following tables, features are marked with the following statuses:
...
+(link:https://issues.redhat.com/browse/RHEL-9279[*RHEL-9279*])
+
+* When you use `preempt-rt` patches with the realtime kernel and you update the SMP affinity of a network interrupt, the corresponding IRQ thread does not immediately receive the update.
```
[the remark this answers is not in the document]
Comment (slovern) [reply release_notes/ocp-4-15-release-notes.adoc:1702]: I think the short form SMP is preferable here

**Review by jldohmann — COMMENTED** [review]
Just a question, otherwise LGTM for merge

File: release_notes/ocp-4-15-release-notes.adoc [hunk 5]
```diff
@@ -2234,6 +2234,24 @@ As a result, if you apply a performance profile, all nodes in the cluster reboot
...
+* When you run Cloud-native Network Functions (CNF) latency tests on an {product-title} cluster, the `oslat` test can sometimes return results greater than 20 microseconds. This results in an `oslat` test failure.
+(link:https://issues.redhat.com/browse/RHEL-9279[*RHEL-9279*])
+
+* When you use `preempt-rt` patches with the real time kernel and you update the SMP affinity of a network interrupt, the corresponding Interrupt Request (IRQ) thread does not immediately receive the update.
+Instead, the update takes effect when the next interrupt is received, and the thread is subsequently migrated to the correct core.
+(link:https://issues.redhat.com/browse/RHEL-9148[*RHEL-9148*])
+
+* The global navigation satellite system (GNSS) module in an Intel Westport Channel e810 NIC that is configured as a grandmaster clock (T-GM) can report the GPS `FIX` state and the GNSS offset between the GNSS module and the GNSS constellation satellites.
```
[the remark this answers is not in the document]
Comment (slovern) [reply release_notes/ocp-4-15-release-notes.adoc:2244]: Hi @jldohmann 
Thanks for the review
Yes the link is on line 2249 below

[the remark this answers is not in the document]
Comment (jldohmann) [reply release_notes/ocp-4-15-release-notes.adoc:2244]: thank you! looks good 😄

[the remark this answers is not in the document]
Comment (slovern) [reply release_notes/ocp-4-15-release-notes.adoc:2244]: thanks again @jldohmann

**Pull request merged**